### PR TITLE
add mini.map

### PR DESCRIPTION
**Pull request by zjp-CN**

### CODE REVIEW

**Comment by phaazon** [comment]
You have conflicts to resolve because other plugins were merged. May I ask you to rebase and resolve them? Thank you a lot.

File: contents/2022/Oct/17.md [hunk 1]
```diff
@@ -20,6 +20,46 @@
...
+- Customizable:
+    - Encoding symbols used to display binary information of different resolution (default is 3x2).
+    - Scrollbar symbols, separate for line and view. Can have any width (even zero, which virtually disables scrollbar).
+    - Integrations producing map line highlights.
+    - Window options: side (left/right), width, 'winblend', and more.
+
+- [Reddit](https://www.reddit.com/r/neovim/comments/y1balh/minimap_window_with_buffer_text_overview/)
+- [mini.map]
+
+[mini.map]: https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-map.md
```
Comment (phaazon) [inline contents/2022/Oct/17.md:59]: Are you sure you can put this here and not at the bottom of the document? I never know that kind of things.

**Review by phaazon — APPROVED** [review]

File: contents/2022/Oct/17.md (hunk 1 above)
Comment (zjp-CN) [reply contents/2022/Oct/17.md:59]: https://www.markdownguide.org/basic-syntax/#reference-style-links

> You can place this second part of the link anywhere in your Markdown document. Some people place them immediately after the paragraph in which they appear while other people place them at the end of the document (like endnotes or footnotes).

Things like `[mini.map]: xxx` can be put anywhere, which is a standard I think.
It seems you use `pulldown-cmark` crate to parse md. I use mdbook a lot which is based on that crate, and it turns out to be fine.

File: contents/2022/Oct/17.md [hunk 2]
```diff
@@ -40,6 +40,45 @@ Copilot Panel, with custom keybindings and configurable settings. It also integr
...
+                <i class="fa-solid fa-book"></i>
+            </span>
+            <span>mini.map</span>
+        </span>
+    </a>
+</h3>
+
+<video controls>
+  <source
+    src="https://user-images.githubusercontent.com/24854248/194914295-2d854c6b-1b5a-418d-9e1b-9e95539ebb7e.mp4"
```
Comment (echasnovski) [inline contents/2022/Oct/17.md:56]:
```suggestion
    src="https://user-images.githubusercontent.com/24854248/195806215-10e05020-50b7-4bee-9447-ee5af3e971ec.mp4"
```

I remade a demo after implementation of new `window.focusable` option. @zjp-CN, would you mind updating the link?

**Review by phaazon — APPROVED** [review]

**Pull request merged**